rdkit/Chem/Draw/SimilarityMaps.py

```diff
@@ -126,14 +127,16 @@ def GetStandardizedWeights(weights):
   return weights, maxAbsWeight
 
 
-def GetSimilarityMapFromWeights(mol, weights, colorMap=None, scale=-1, size=(250, 250), sigma=None,
-                                coordScale=1.5, step=0.01, colors='k', contourLines=10, alpha=0.5,
-                                draw2d=None, **kwargs):
+def GetSimilarityMapFromWeights(mol, weights, draw2d, colorMap=None, scale=-1, size=(250, 250),
```

Comment:
A doc test is broken now:   GetSimilarityMapFromWeights() that uses draw2d

Reply:
Ok, I'm confused why the doc test is broken, it's using the older api:

File "GettingStartedInPython.rst", line 2404, in default
Failed example:
    fig = SimilarityMaps.GetSimilarityMapFromWeights(mol, contribs, colorMap='jet', contourLines=10)
Exception raised:
    Traceback (most recent call last):
      File "/usr/share/miniconda/envs/rdkit_build/lib/python3.8/doctest.py", line 1336, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest default[4]>", line 1, in <module>
        fig = SimilarityMaps.GetSimilarityMapFromWeights(mol, contribs, colorMap='jet', contourLines=10)
    TypeError: GetSimilarityMapFromWeights() missing 1 required positional argument: 'draw2d'

Reply:
Scratch that, the draw2d is now required so we need to update the rdkit book

Reply:
> Scratch that, the draw2d is now required so we need to update the rdkit book

Yep, I'm on it. I had forgotten to check the doctests

Reply:
@bp-kelley this problem is cleared up and the builds are working again